Kickstarter-iOS/Views/Controllers/ProjectCreatorViewController.swift

```diff
@@ -33,6 +33,9 @@ internal final class ProjectCreatorViewController: WebViewController {
 
     _ = self
       |> baseControllerStyle()
+
+    _ = self.navigationController?.navigationBar
+      ?|> baseNavigationBarStyle
```

Comment:
i have an idea to be able to bake `baseNavigationBarStyle` into our `baseControllerStyle` so we wouldn't have to do any of this manually. lemme know if you wanna pair on it today